.travis.yml

```diff
@@ -5,23 +5,7 @@ matrix:
     - script:
         - bin/fetch-configlet
         - bin/configlet lint .
-        - docker run -v `pwd`:/swift swift:4.0.2 bash -c "cd /swift && ./xswift-test-spm"
-      env: JOB=Linux40
-      sudo: required
-      services: docker
-    - script:
-        - bin/fetch-configlet
-        - bin/configlet lint .
-        - docker run -v `pwd`:/swift norionomura/swift:swift-4.1-branch bash -c "cd /swift && ./xswift-test-spm"
-      env: JOB=Linux41x
-      sudo: required
-      services: docker
-    - script:
-        - bin/fetch-configlet
-        - bin/configlet lint .
-        - docker run -v `pwd`:/swift norionomura/swift:swift-4.2-branch bash -c "cd /swift && swift test --verbose"
```

Comment:
This one tests the root package. We would like to test all individual packages and the root packages which calls into all the subpackages.